proto/spec/api_collection.proto

```diff
@@ -49,6 +49,7 @@ message ParseDirective {
   string function_template = 2;
   BlockParser result_parsing = 3 [(gogoproto.nullable) = false];
   string api_name = 4;
+  string expected_result = 5;

```

Comment:
let's use existing field default_value just because we can

Reply:
Fixed 3b283cc485803420e643319f50bead02960b67c3